src/get-started/java-binding.md

```diff
@@ -7,7 +7,7 @@ and Java services runtime environment.
 
 To run a node with your Java service you need to use Exonum Java application.
 You can download an archive containing the application and all the necessary
-dependencies on [GitHub releases page][github-releases]. We suggest to use
+dependencies on [the Releases page][github-releases] on GitHub. We suggest using
 `debug` version during development and `release` for production builds.
```

Comment:
for deployment? The app does not affect builds in any way (except integration tests :upside_down_face: ).

Reply:
Well, makes sense